fmcapi/api_objects.py

```diff
@@ -4906,84 +4906,243 @@ def destination_port(self, action, name=''):
                 del self.destinationPorts
                 logging.info('All Destination Ports removed from this ACPRule object.')
 
-    def source_network(self, action, name=''):
+    def source_network(self, action, name='', literal=dict()):
```

Comment:
Mutable dict() should not be used as an argument in the method because it will only be evaluated when then method is first created, not on each subsequent use of the method. As a result the mutated literal will be unexpectedly present in subsequent uses (First gotcha: https://docs.python-guide.org/writing/gotchas/)